docker/debian-aarch64/Dockerfile

```diff
@@ -33,18 +48,18 @@ RUN command curl -sSL https://rvm.io/mpapis.asc | gpg --import - && \
     command curl -sSL https://rvm.io/pkuczynski.asc | gpg --import - && \
     curl -L -sSL https://get.rvm.io | bash -s stable
 RUN /bin/bash -l -c "rvm requirements"
-RUN /bin/bash -l -c "rvm install 2.6.5"
-RUN /bin/bash -l -c "gem install bundler -v 1.17.3 --no-document"
+RUN /bin/bash -l -c "rvm install 3.0.6"
+RUN /bin/bash -l -c "gem install bundler -v 2.2.3 --no-document"
 
 # pre-load the omnibus dependencies
-RUN git clone https://github.com/rapid7/metasploit-omnibus.git
-RUN cd metasploit-omnibus && /bin/bash -l -c "bundle install --binstubs"
-RUN rm -fr metasploit-omnibus
+RUN /bin/bash -l -c 'cd ~/ && git clone https://github.com/rapid7/metasploit-omnibus.git && \
+        cd ~/metasploit-omnibus && bundle install && bundle binstubs --all && cd ~/ && rm -fr metasploit-omnibus'
 
 ENV JENKINS_HOME /home/jenkins
-RUN useradd -d "$JENKINS_HOME" -u 1001 -m -s /bin/sh jenkins
+RUN id jenkins || useradd -d "$JENKINS_HOME" -u 1001 -m -s /bin/sh jenkins
```

Comment:
idempotent user add since we're still extending the previous rapid7/msf-debian-aarch64-omnibus:2020_03 - but might not in the future